metadata.rb

```diff
@@ -19,10 +19,11 @@
 # build-essential is obsolete in chef 14+
 depends 'build-essential' # ~FC121
 depends 'nssm', '>= 3.42.0'
-depends 'golang'
+depends 'golang', '< 5.0.0'
```

Comment:
< constraints should not be added here, if it breaks the tests, add it to Berksfile.
Else this cookbook may be compatible with Chef 14 while we restrict golang for Chef 14.